OneSignalSDK/unittest/src/test/java/com/test/onesignal/LocationIntegrationTests.java

```diff
@@ -48,6 +48,7 @@
 import org.junit.runner.RunWith;
 import org.robolectric.Robolectric;
 import org.robolectric.RobolectricTestRunner;
+import org.robolectric.RuntimeEnvironment;
 import org.robolectric.android.controller.ActivityController;
 import org.robolectric.annotation.Config;
 import org.robolectric.shadows.ShadowApplication;
```

Comment:
Can we remove this ShadowApplication  depedency?

Reply:
Good catch, Fixed!